### ISSUE TITLE
Prompt has no focus after starting

### ISSUE THREAD

**Issue by elementaryBot** [elementary/terminal#66]
After starting Pantheon Terminal with Slingshot (keyboard only, but sometimes with mouse, too) the terminal is opened, but has no focus. So you have to click additionally into it to enter some commands.

Launchpad Details: [#LP1624741](https://bugs.launchpad.net/bugs/1624741) Daniel - 2016-09-17 21:14:24 +0000

**Comment by elementaryBot** [comment]
Hey,

i don't think, that this is the same issue.
pantheon-terminal is opened in foreground and top most app. But the prompot is not in focus.

At least this problem only occures, when I open it with slingshot.
Furthermore, other terminals like terminix, gnome-terminal do have the focus.

Launchpad Details: [#LPC](https://bugs.launchpad.net/pantheon-terminal/+bug/1624741/comments/1) Daniel - 2016-09-20 20:23:39 +0000

**Comment by elementaryBot** [comment]
Apologies, then.

Launchpad Details: [#LPC](https://bugs.launchpad.net/pantheon-terminal/+bug/1624741/comments/2) Zisu Andrei - 2016-09-20 20:38:13 +0000

**Comment by FredrikAugust** [comment]
Bump, I have the same issue when launching from slingshot.

**Comment by jeremypw** [comment]
I think this will be fixed by 1f3903cc1cd9a6005ed9c6b0bdd7338b6f60b8be, which is not yet released to stable.

**Comment by peteruithoven** [comment]
I actually think this is a Applications menu issue. I'm closing this in favor of: https://github.com/elementary/applications-menu/issues/19
